plugin-enclave-gradle/build.gradle

```diff
@@ -158,11 +158,23 @@ processResources {
     }
 }
 
+def genDockerContainerTag() {
+    new ByteArrayOutputStream().withStream { os ->
+        exec {
+            executable = 'bash'
+            args "-c", "../scripts/genDockerImageTag.sh" // N.B.: The script must only print to the standard output
```

Comment:
Not a big deal, but can we avoid to have this relative `../` way of calling the script? I see that we used `projectDir` elsewhere, I am wondering if it is nicer to be consistent.

Reply:
`${rootProject.projectDir}`